source/includes/api/_dataset.md

```diff
@@ -2,72 +2,13 @@
 
 ## What is a Dataset?
 
-A dataset abstracts the data that can be obtained from several sources into a common interface. There are several data providers supported in the API, and each of those has a different provider. Datasets can belong to several applications.
+If you are new to the RW API, or want to learn more about the concept of a dataset, we strongly encourage you to read the [dataset concept](#dataset) documentation first. It gives you a brief and clear description of what a dataset is, and what it can do for you. 
```

Comment:
Aren't we renaming it to "WRI API"?

Reply:
If we are, then it should be done in a separate PR, and it needs to be changed all over the docs